src/citrine/resources/team.py

```diff
@@ -33,6 +33,52 @@ def __str__(self):
             .format(self.user.screen_name, self.actions, self.team.name)
 
 
+class TeamResourceIDs:
+    """
+    A Citrine Team Resource IDs class.
+
+    This class is used to list the unique IDs for specific resource
+    types published in a single team and therefore available to be pulled
+    in by all projects.
+
+    Parameters
+    ----------
+    team_id: str or uuid
+        ID of the team.
+    resource_type: str
+        The resource type to list, one of DATASET/MODULE/TABLE/TABLE_DEFINITION
+
+    """
+
+    _api_version = "v3"
+
+    def __init__(self,
+                 session: Session,
+                 team_id: Union[str, UUID],
+                 resource_type: str) -> None:
+        self.session = session
+        self.team_id = team_id
+        self.resource_type = resource_type
+
+    def _path(self) -> str:
+        return format_escaped_url(f'/teams/{str(self.team_id)}')
```

Comment:
I believe the f-string automatically casts as `str` by default
```suggestion
        return format_escaped_url(f'/teams/{self.team_id}')
```